softwareComponents/configuration/src/rofibot.cpp

```diff
@@ -20,6 +20,13 @@ double roficom::orientationToAngle( roficom::Orientation o ) {
     return 0; // never reached
 }
 
+Matrix roficom::orientationToTransform(roficom::Orientation orientation) {
```

Comment:
Missing spaces within brackets.

Reply:
Addressed in [4eb8bdd...1ac00c2](https://github.com/paradise-fi/RoFI/compare/4eb8bddc3e832cfe0224124baa72ddffc874d582..1ac00c2ee26269312e7b13afced9d5cfe4f22cda)